pkg/envoy/lds/response_test.go

```diff
@@ -119,10 +119,16 @@ func TestNewResponse(t *testing.T) {
 	assert.Len(listener.ListenerFilters, 3) // Test has egress policy feature enabled, so 3 filters are expected: OriginalDst, TlsInspector, HttpInspector
 	assert.Equal(listener.ListenerFilters[0].Name, wellknown.OriginalDestination)
 	assert.NotNil(listener.FilterChains)
-	// There are 3 filter chains configured on the outbound-listner based on the configuration:
+	// There are 4 filter chains configured on the outbound-listener based on the configuration:
```

Comment:
It seems to me they are still only 3 filter chains.

Reply:
Good catch, updated the comment!